crontainer/task.go

```diff
@@ -14,7 +14,7 @@ type Task struct {
 
 func NewTask(command string, schedule string, name string) *Task {
 	if name == "" {
-		name = randStringBytes(15)
+		name = uuid.NewV4()
```

Comment:
This will not work. `name` is a `string`, but `New4()` returns a `UUID`. You can write `uuid.NewV4().String()`.